modules/migrations/base/error.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2021 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package base
+
+import "fmt"
+
+// ErrNotSupported represents status if a downloader do not supported something.
+type ErrNotSupported struct {
+	Entity string
+}
+
+// IsErrNotSupported checks if an error is an ErrNotSupported
+func IsErrNotSupported(err error) bool {
+	_, ok := err.(ErrNotSupported)
```

Comment:
Seems incorrect here, because the code returns pointer: `return nil, &ErrNotSupported{}`

Reply:
Will be fixed in 
* #20182